src/libs/ReportActionsUtils.ts

```diff
@@ -1200,7 +1201,7 @@ function getMessageOfOldDotReportAction(reportAction: OnyxEntry<ReportAction>):
     if (!Array.isArray(reportAction?.message)) {
         return getReportActionText(reportAction);
     }
-    return reportAction?.message?.map((element) => getTextFromHtml(element?.html)).join('') ?? '';
+    return reportAction?.message?.map((element) => getTextFromHtml(element?.html ?? element?.text)).join('') ?? '';
```

Comment:
This is safer given we have `html` as '' in one of the cases. Also, please add a comment after using || here and in getReportActionText specifying why we are using || against ??

```suggestion
    return reportAction?.message?.map((element) => getTextFromHtml(element?.html || element?.text)).join('') ?? '';
```